assets/scss/components/_modal.scss

```diff
@@ -73,6 +72,18 @@ $modal-width: 440px;
 	z-index: map-get($zindex-map, "popup") + 1;
 }
 
+.modal-closeButtonContainer--sticky {
+	background-color: $C_contentBG;
+	border-bottom: 1px solid $C_border;
+	position: sticky;
+	position: -webkit-sticky;
```

Comment:
Should the vendor-prefixed value come _before_ the standard in this case?

Reply:
I think both work, but putting it before feels better in this case